testutils/mocks/Logger.go

```diff
@@ -0,0 +1,81 @@
+package mocks
```

Comment:
Is this auto gen'd code? Is it possible to put a note here about that? Or somewhere?